sklearn/kernel_approximation.py

```diff
@@ -444,6 +445,12 @@ class SkewedChi2Sampler(
     1.0
     """
 
+    _parameter_constraints = {
+        "skewedness": [Real],
+        "n_components": [Integral],
```

Comment:
[`utils._param_validation.Interval`](https://github.com/scikit-learn/scikit-learn/blob/b3fe9b8a5316b1a89e3bed37c989121a52bb7db5/sklearn/utils/_param_validation.py#L321-L352)s have to be specified for `Real` and `Integral` parameters which bounds informations.

For instance, see usage of `utils._param_validation.Interval` in context:

https://github.com/scikit-learn/scikit-learn/blob/b3fe9b8a5316b1a89e3bed37c989121a52bb7db5/sklearn/decomposition/_nmf.py#L912-L946

Reply:
Updated!